ivy/functional/frontends/paddle/math.py

```diff
@@ -193,6 +193,11 @@ def divide(x, y, name=None):
 def erf(x, name=None):
     return ivy.erf(x)
 
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def erfinv(x, name=None):
+    return  ivy.erfinv
```

Comment:
you are returning a function reference here. Do you mean to do `ivy.erfinv(x)` here ( provided that `ivy.erfinv` exists)?

Reply:
Yea my bad let me fix it, also I think ivy.erfinv does not exist as of now : [erfinv-functional](https://github.com/unifyai/ivy/issues/23685). So is there a way to bypass it as someone is already working to implement erfinv in functional.